lib/src/components/avatars/avatar.dart

```diff
@@ -316,6 +316,17 @@ extension on ZetaAvatarSize {
     }
   }
 
+  static double fontSize(BuildContext context, ZetaAvatarSize size) {
```

Comment:
If we are adding this here we should add documentation

Reply:
I've added `/// Font size for initials` above this function. Is that okay or should there be more description?